go/src/step1_read_print/step1_read_print.go

```diff
@@ -1,37 +1,35 @@
 package main
+
 // import "bufio"
 import (
-        "fmt"
-        "bufio"
-        "os"
-        "reader"
+	"fmt"
+	"reader"
 )
 
 func main() {
-  scanner := bufio.NewScanner(os.Stdin)
-  fmt.Print("user> ")
+	fmt.Print("user> ")
 
-  for scanner.Scan() {
-    line := scanner.Text()
-    fmt.Println(rep(line));
-    fmt.Print("user> ")
-  }
+	for {
+		line := "( 1 2 3)"
+		fmt.Println(rep(line))
+		fmt.Print("user> ")
```

Comment:
Again, just for debugging. Using `delve` I was not able to run a debuggin session AND  give input to the program, so I just made it loop over this silly string.